app/src/pages/inside/logsPage/logItemInfo/logItemInfoTabs/logItemInfoTabs.jsx

```diff
@@ -99,31 +109,31 @@ export class LogItemInfoTabs extends Component {
     isSauceLabsIntegrationView: PropTypes.bool.isRequired,
     sauceLabsIntegrations: PropTypes.array.isRequired,
     fetchFirstAttachments: PropTypes.func,
+    setActiveTabId: PropTypes.func,
     attachments: PropTypes.array,
     tracking: PropTypes.shape({
       trackEvent: PropTypes.func,
       getTrackingData: PropTypes.func,
     }).isRequired,
+    activeTabId: PropTypes.string,
   };
 
   static defaultProps = {
     lastActivity: null,
     fetchFirstAttachments: () => {},
     attachments: [],
+    activeTabId: 'logs',
+    setActiveTabId: () => {},
   };
 
   static getDerivedStateFromProps(props) {
     return props.loading
       ? {
-          activeTabId: null,
+          activeTabId: 'logs',
```

Comment:
- When the page is reloaded, the active tab is not saved, it is always a `logs`. Was it by design?

- It is also better to take this value into a constant and use it.

Reply:
I returned 'null'

Reply:
I added this point as a comment inside of the task. It should be discussed.